session/bootstrap.go

```diff
@@ -396,6 +396,23 @@ const (
 		UNIQUE KEY table_version (table_id, version),
 		KEY table_create_time (table_id, create_time)
 	);`
+	// CreateAnalyzeJobs stores the analyze jobs.
+	CreateAnalyzeJobs = `CREATE TABLE IF NOT EXISTS mysql.analyze_jobs (
+		id BIGINT(64) UNSIGNED NOT NULL AUTO_INCREMENT,
+		update_time TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+		table_schema CHAR(64) NOT NULL DEFAULT '',
+		table_name CHAR(64) NOT NULL DEFAULT '',
+		partition_name CHAR(64) NOT NULL DEFAULT '',
+		job_info TEXT NOT NULL DEFAULT '',
+		processed_rows BIGINT(64) UNSIGNED NOT NULL DEFAULT 0,
+		start_time TIMESTAMP,
+		end_time TIMESTAMP,
+		state ENUM('pending', 'running', 'finished', 'failed') NOT NULL,
+		instance CHAR(64) NOT NULL comment 'address of the TiDB instance executing the analyze job',
+		process_id BIGINT(64) UNSIGNED comment 'ID of the process executing the analyze job',
```

Comment:
Seems we plan to record the connection ID for `analyze` initiated by the client and record process ID for auto analyze.
I think the column name and the comment is a bit confusing.

Reply:
The ID is the same as `ID` in `show processlist`(#31724 makes auto-analyze appear in `show processlist`). #31724 uses the fixed and reserved ID for auto analyze.
https://github.com/pingcap/tidb/blob/164bee7ae88ab5bb28ef6e6528bd6963f4fb7ba2/util/processinfo.go#L265-L274
And the ID is used to set `GetSessionVars().ConnectionID` of the internal session which is going to execute auto-analyze.
https://github.com/pingcap/tidb/blob/164bee7ae88ab5bb28ef6e6528bd6963f4fb7ba2/domain/domain.go#L1812-L1823